### ISSUE TITLE
MKV Support - Current Status

### ISSUE THREAD

**Issue by paulwaldmann** [kaltura/nginx-vod-module#1272]
I read that MKV support is discontinued, meaning we can't use it anymore, or is there some other way to use MKV?

My client gets a lot of his content in AVC/E-AC-3 but it's in an MKV container.

Maybe converting it first with copy param like this, `ffmpeg -i input.mkv -c copy output.mp4` would work?

Either way, thank you for this underrated repo 💯👍🏻

**Comment by erankor** [comment]
Yes, MKV support was added in order to support VP9 codec, but some time after it was added, ffmpeg added support for VP9 in MP4 container. So we didn't really have a need for this feature, and I never finalized it. 

It can be enabled by uncommenting this line -
https://github.com/kaltura/nginx-vod-module/blob/master/ngx_http_vod_module.c#L256
But I know there are issues there, for some files it will work, for some it won't...

I would recommend repackaging to MP4 (without re-transcoding), as done with the sample command you pasted.
